### ISSUE TITLE
Segment data is lower than expected

### ISSUE THREAD

**Issue by unn** [drud/ddev#2210]
**Describe the bug**
Looking at segment data, it seems that following the v1.13.0 on 4 Feb, we've seen a decrease in data across the board.

**Comment by unn** [comment]
> Do you ever get "Instrumentation is opted in, but SegmentKey is not available." when you issue #ddev commands? If so, could you PM me the output of ddev --version && ddev config global? Thanks!
https://twitter.com/randyfay/status/1252630835536130049

**Comment by rfay** [comment]
One person with 1.13.1 reported that they got the "SegmentKey is not available". It was macOS/Homebrew. The size of their binary and the checksum did *not* match other homebrew installs. The history of this is unknown, but it could indicate a process or CI problem. Certainly if that happened to very many people, it would cause the segment data concern.

**Comment by rfay** [comment]
I'm going to close this for now. Most of our indicators seem to imply that we're getting what we should probably expect to get. Happy to reopen, and of course our ears will be very much open for issues.

**Comment by dev-sukhjinder** [comment]
I just upgraded to latest ddev version, details are below

[https://www.dropbox.com/s/w6kehfha02vmp68/Screenshot%202021-06-04%20at%2012.35.54%20PM.png?dl=0](url)

> DDEV-Local version	v1.17.3
architecture      	amd64
db                	drud/ddev-dbserver-mariadb-10.3:v1.17.3
dba               	phpmyadmin:5
ddev-ssh-agent    	drud/ddev-ssh-agent:v1.17.0
docker            	19.03.13
docker-compose    	1.27.4
os                	darwin
router            	drud/ddev-router:v1.17.3
web               	drud/ddev-webserver:v1.17.3

When I run and ddev command I get the following message

```
$ ddev drush cr
[success] Cache rebuild complete.
Instrumentation is opted in, but SegmentKey is not available.
```

**Comment by rfay** [comment]
Thanks so much for catching that @dev-sukhjinder. It's a build problem (this is the first release to use GitHub Actions instead of CircleCI for the release). I'll fix this in a v1.17.4, probably today.
